### PR TITLE
Update IRC-Bot.md

### PR DESCRIPTION
**Pull request by allano337**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

In the doc's doesn't it say to run ./irc.php to start the IRC-Bot, I have added it to the doc's

Also one of our engineers made a very basic systemd start up script in /etc/systemd/system/librenms-irc.service set to start at boot but it can be stopped or started just like any other systemd script such as systemctl start librenms-irc.service

librenms-irc.service
[Unit]
Description=Librenms IRC bot

[Service]
ExecStart=/opt/librenms/irc.php
Restart=always

[Install]
WantedBy=default.target

laf suggested that we should contribute this back to the code base by submitting a pull request.

Thanks

### CODE REVIEW

**Comment by laf** [comment]
@allano337 if you can add that file into the misc/ folder and reference it in the doc you've edited. That would be awesome.

File: doc/Extensions/IRC-Bot.md [hunk 1]
```diff
@@ -5,13 +5,13 @@ Table of Content:
    -   [Commands](#commands)
 -   [Examples](#examples)
 -   [Extensions](#extensions)
-
+-   Systemd start up script
```
Comment (murrant) [inline doc/Extensions/IRC-Bot.md]: This should be a link like the line above it.

File: doc/Extensions/IRC-Bot.md [hunk 2]
```diff
@@ -143,3 +143,21 @@ File: includes/ircbot/echo.inc.php
       return $this->respond("root shouldn't be online so late!");
    }
 ```
+
+Systemd start up script
```
Comment (murrant) [inline doc/Extensions/IRC-Bot.md]: This should be a heading, see other headings for examples.

File: doc/Extensions/IRC-Bot.md [hunk 3]
```diff
@@ -143,3 +143,21 @@ File: includes/ircbot/echo.inc.php
...
+
+Basic systemd start up script to be placed in /etc/systemd/system/ to start irc service at boot. 
+
+librenms-irc.service script is located at /opt/librenms/misc/
+
+Once copied to /etc/systemd/system/ you must run the following commads:
```
Comment (murrant) [inline doc/Extensions/IRC-Bot.md]: commads -> commands

**Pull request merged**